pkg/activator/throttler.go

```diff
@@ -135,7 +141,11 @@ func (t *Throttler) UpdateCapacity(rev RevisionID, size int) error {
 // It returns an error if either breaker doesn't have enough capacity,
 // or breaker's registration didn't succeed, e.g. getting endpoints or update capacity failed.
 func (t *Throttler) Try(ctx context.Context, rev RevisionID, function func()) error {
-	breaker, existed := t.getOrCreateBreaker(rev)
+	revision, err := t.revisionLister.Revisions(rev.Namespace).Get(rev.Name)
```

Comment:
It feels wrong calling this on the hot-path. Should this be moved into the `getOrCreateBreaker` method and only execute it if the breaker doesn't exist yet? That'd cause a duplicate lookup above in `UpdateCapacity` but that's not on the hot-path.

If we'd want to alleviate that, we could actually cache the containerConcurrency value in the breaker's map I suppose. It's immutable anyway.

Reply:
Second is what I was going to do at first :-)

Reply:
It's basically map lookup, but sure, let's put it internally. Though, this adds also an error return :|